x-pack/plugin/inference/src/yamlRestTest/resources/rest-api-spec/test/inference/10_semantic_text_field_mapping.yml

```diff
@@ -63,6 +63,60 @@ setup:
   - match: { "test-index.mappings.properties.sparse_field.model_settings.task_type": sparse_embedding }
   - length: { "test-index.mappings.properties.sparse_field": 3 }
 
+---
+"Add field_caps support for semantic_text field types with sparse embedding":
```

Comment:
The test can only work on 8.16 so you'll need to skip other versions with something like:

```
  - requires:
      cluster_features: "gte_v8.16.0"
      reason: field_caps support for semantic_text added in 8.16.0
```

Reply:
The name of the test doesn't need to be that verbose, "Field caps with sparse embedding" should be enough ;)